src/js/actions/DialogActions.js

```diff
@@ -1,72 +1,84 @@
+var Util = require("../helpers/Util");
 var AppDispatcher = require("../AppDispatcher");
 var DialogEvents = require("../events/DialogEvents");
+var DialogTypes = require("../constants/DialogTypes");
+var dialogScheme = require("../stores/schemes/dialogScheme");
 
 var DialogActions = {
-  alert: function (message, dismissButtonLabel = "OK") {
-    const dialogId = Symbol(message);
-    AppDispatcher.dispatchNext({
-      actionType: DialogEvents.ALERT_SHOW,
-      dialogId: dialogId,
-      dismissButtonLabel: dismissButtonLabel,
-      message: message
-    });
-    return dialogId;
-  },
-  alertDismiss: function (dialogId) {
-    AppDispatcher.dispatchNext({
-      actionType: DialogEvents.ALERT_DISMISS,
-      dialogId: dialogId
-    });
-  },
-  confirm: function (message, successButtonLabel = "OK") {
-    const dialogId = Symbol(message);
+  dialog: function (data) {
```

Comment:
Exactly, I like to stick with the aliases as they are now. It's much more expressive and convenient. So to avoid confusion and also force consistent usage I would make ```showDialog()``` private.